src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/ForwardCancellationTokenToInvocationsTests.cs

```diff
@@ -2329,6 +2329,48 @@ static async Task M1Async(string s, CancellationToken cancellationToken)
             return VerifyCS.VerifyCodeFixAsync(originalCode, fixedCode);
         }
 
+        [Fact]
+        [WorkItem(4842, "https://github.com/dotnet/roslyn-analyzers/issues/4842")]
+        public Task CS_ParamsArray()
+        {
+            // Local static functions are available in C# >= 8.0
```

Comment:
Copy/paste error?

Looks like some existing tests in `main` has this unrelated comment as well :smile:

```suggestion
```